packages/sdk/src/swap/types.ts

```diff
@@ -80,6 +80,7 @@ interface DepositAddressFields extends SwapStatusResponseCommonFields {
   estimatedDepositChannelExpiryTime: number;
   isDepositChannelExpired: boolean;
   depositChannelOpenedThroughBackend: boolean;
+  affiliateBrokers?: { account: string; commissionBps: number }[];
```

Comment:
not sure if we should call this `beneficiaries` because it will also include the submitter? or maybe it wont?

Reply:
can talk about this in person

Reply:
We don't have access to the main broker, so I've renamed it to affiliates

Reply:
hm would it be better if we include all beneficiaries (like in https://github.com/chainflip-io/chainflip-web-services/pull/1571)? i think we could get the submitter from the event, right?

Reply:
I think it makes more sense for the response format to match the request format. It's possible to get the submitter, but I don't know if there is a use case for including it. Odds are the person requesting a status with the SDK knows the broker used to create the deposit channel. If we get feedback that it would be useful, we can definitely include it. But at this stage I think it makes more sense to keep it simple and uniform with the request format